src/components/Table/TableRow.tsx

```diff
@@ -91,13 +91,22 @@ export class TableRow<T> extends React.PureComponent<TableRowProps<T>, {}> {
 						typeof column.cellAttributes === 'function'
 							? column.cellAttributes(data, get(data, column.field))
 							: column.cellAttributes || {};
+					let url: URL | null;
+					try {
+						url = new URL(href ?? '');
+					} catch (err) {
+						url = null;
+					}
```

Comment:
I'm not too sure, but I would probably use something like:
```
const isInternalUrl = !href.startsWith('http')
```

so we can avoid the try catch and new URL function